### DISCUSSION TITLE
Export / Import of trilium data (Feature request)

### DISCUSSION THREAD

**Discussion by ajcallegari** [TriliumNext/Trilium#7334] [Q&A] [answered]
### Describe feature

Would be great to have easy export and import of the database, either into a different file format or into a zip archive, for the purpose of backing up the data or moving it from one computer to another. Currently I have no clue how to back the SQLite db to google drive.  There's not even a way to find the location of the db on my Mac OS desktop app (no, there is no report of the data location under Trilium Notes -> About Trilium Notes).   Currently I'm very concerned that the db will be corrupted by a crash or computer will fail and all of my Trilium data will be lost.

**Comment by sigaloid** [comment, marked as answer]
You can export by right-clicking the root node (it's recursive). You can then select a number of options to export it, including a ZIP

**Comment by meichthys** [comment]
https://github.com/zadam/trilium/wiki/Data-directory

**Comment by ajcallegari** [comment]
Thanks for pointing that out!  HTML export/import is just what I needed. Maybe this functionality is a bit hidden and could be exposed more clearly for the user by putting it on the File pull-down menu (File -> Export Node) where export functionality is usually made available?

**Comment by pmorch** [comment]
What specifically is left to be done for this issue? Improve the discoverability of export, e.g. by putting int in "File --> Export Node"?

FWIW, I'm investigating exporting, and I've come across multiple search hits mentioning right-clicking the root node and hitting export, so for me there was no discovery problem.

Or can this issue be closed?

**Reply by eliandoran** [reply]
I believe we can close it.
